### ISSUE TITLE
allow php tags at end of line without screwing up indentation

### ISSUE THREAD

**Issue by mbtuckersimm** [fxbois/web-mode#1089]
The house style at my workplace is to place `<?php` and `?>` tags at the end of lines, like so:
```
<?php
$x = 5; ?>
<span>The number is <?php echo $x; ?></span> <?php
$x = 7;
```
 
This interacts badly with auto-indent: if I press `TAB` to indent with point on the `$x = 7` line, I end up with this:
```
<?php
$x = 5; ?>
<span>The number is <?php echo $x; ?></span> <?php
                                             $x = 7;
```
Is there some way to have the PHP block not indent to the same level as the preceding `<?php` tag?

**Comment by fxbois** [comment]
@mbtuckersimm you can now use the value ```-1``` for ```web-mode-block-padding```

**Comment by mbtuckersimm** [comment]
Thank you very much for addressing this.

Is this on melpa yet or do I need to clone the repo to get this feature? I updated `web-mode` from the package menu and I'm now running version 20191028.2050, but it's not working for me. (I am using emacs 26.3 on linux mint 19.2, if that's relevant.)

I tried using the customize menu to set `web-mode-block-padding`, but the option `-1` is not available in the value menu. Then I tried setting it in my `init.el` by adding `(web-mode-block-padding -1)` to the `:custom` section of my `use-package` form for web-mode, but the indentation behavior still seems to be the same as before.

Your help is much appreciated.
